src/audio-worklet/free-queue/examples/hello-webassembly/build/example.worker.js

```diff
@@ -0,0 +1,159 @@
+/**
```

Comment:
Is this generated by Emscripten?

Reply:
Yup!
Everything in build folder is generated by Emscripten.